results/results-app.js

```diff
@@ -0,0 +1,15 @@
+import { getChar } from '../utils.js';
+
+
+
+const results = document.querySelector('h1');
+
+const character = getChar();
+
+if (character.hp <= 0) {
+    results.textContent = 'YOU DIED!!';
+} if (character.hp > 0 && character.completed.room5 === true) {
```

Comment:
think you might be looking for an else here, but seems regardless, your user is dying through every choice.